queue/src/main/java/com/bazaarvoice/emodb/queue/core/kafka/KafkaConfig.java

```diff
@@ -0,0 +1,129 @@
+import com.amazonaws.services.simplesystemsmanagement.model.Parameter;
+import org.apache.kafka.clients.admin.AdminClientConfig;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+public class KafkaConfig {
+
+    private static final Logger logger = LoggerFactory.getLogger(KafkaConfig.class);
+
+    // Static SSM Client and configuration using AWS SDK v1
+    private static final AWSSimpleSystemsManagement ssmClient = AWSSimpleSystemsManagementClientBuilder
+            .standard()
+            .build();
+
+    private static final String DEFAULT_BOOTSTRAP_SERVERS =
+            "b-1.qaemodbpocmsk.q4panq.c10.kafka.us-east-1.amazonaws.com:9092," +
+                    "b-2.qaemodbpocmsk.q4panq.c10.kafka.us-east-1.amazonaws.com:9092";
+
+    private static String bootstrapServersConfig;
+    private static String batchSizeConfig;
+    private static String retriesConfig;
+    private static String lingerMsConfig;
+
+    static {
+        try {
+            // Load configurations from SSM during static initialization
+            Map<String, String> parameterValues = getParameterValues(
+                    Arrays.asList(
+                            "/emodb/kafka/batchSize",
+                            "/emodb/kafka/retries",
+                            "/emodb/kafka/lingerMs",
+                            "/emodb/kafka/bootstrapServers"
+                    )
+            );
+
+            // Set configurations with fallback to defaults if not present
+            // Sets the batch size for Kafka producer, which controls the amount of data to batch before sending.
+            batchSizeConfig = parameterValues.getOrDefault("/emodb/kafka/batchSize", "16384");
+
+            // Sets the number of retry attempts for failed Kafka message sends.
+            retriesConfig = parameterValues.getOrDefault("/emodb/kafka/retries", "3");
+
+            // Sets the number of milliseconds a producer is willing to wait before sending a batch out
+            lingerMsConfig = parameterValues.getOrDefault("/emodb/kafka/lingerMs", "1");
+
+            // Configures the Kafka broker addresses for producer connections.
+            bootstrapServersConfig = parameterValues.getOrDefault("/emodb/kafka/bootstrapServers", DEFAULT_BOOTSTRAP_SERVERS);
+
+            logger.info("Kafka configurations loaded successfully from SSM.");
+        } catch (AmazonServiceException e) {
+            logger.error("Failed to load configurations from SSM. Using default values.", e);
+            throw e;
+        }
+        catch (Exception e) {
+            logger.error("Unexpected error occurred while loading configurations from SSM. Using default values.", e);
```

Comment:
If the exception is thrown, why to log it here? its unnecessary logging and discomfort in debugging. 
todo: Make the last catch block in the trace log it before acting on the exception. This is a logging best practice